src/sys/select.rs

```diff
@@ -53,6 +53,25 @@ impl FdSet {
             *bits = 0
         }
     }
+
+    /// Finds the largest file descriptor in the set.
```

Comment:
I don't think `largest` is an appropriate term here, as `RawFd`s don't have a size. They have a numbering, however, so `highest` might be appropriate here. Also, since this basically exists exclusively for use with `select()`, please add a simple example showing how you might use it here.